From e81967066a4ebc69bb4e9712541f20ce2436fa88 Mon Sep 17 00:00:00 2001
From: Igor Grinberg <igrinberg@habana.ai>
Date: Tue, 21 Apr 2020 17:10:54 +0300
Subject: [PATCH] [SW-10446] preboot: add message before cmds loop

This message is important as w/o it one can think the Preboot is stuck.

Change-Id: I6b155cff4c6ab82a17848033dcd21ab945b13298
Signed-off-by: Igor Grinberg <igrinberg@habana.ai>
---

diff --git a/app/preboot.c b/app/preboot.c
index 8c3358b..eed483d 100644
--- a/app/preboot.c
+++ b/app/preboot.c
@@ -66,6 +66,7 @@
 	for (j = 0; j < size; j++)
 		cmd_ops[j].setup();
 
+	preboot_puts("Waiting for commands...\n");
 	i = 0;
 	while (1) {
 		res = cmd_ops[i].check();
